cmd/docs/docs.go

```diff
@@ -0,0 +1,62 @@
+package docs
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/cobra/doc"
+)
+
+type Options struct {
+	// the path to generate the documentation in
+	path string
+}
+
+const DefaultPath = "./docs"
+
+func NewDocsCommand() *cobra.Command {
+	opts := &Options{}
+
+	return &cobra.Command{
+		Use:   "docs",
+		Short: "Generates the documentation for the CLI",
+		Args:  cobra.MaximumNArgs(1),
+		RunE: func(cmd *cobra.Command, args []string) error {
```

Comment:
I'd go ahead and get rid of the `###### Auto generated by spf13/cobra on 5-Sep-2024` auto generated bit on the tail end of the markdown docs with:

```suggestion
		RunE: func(cmd *cobra.Command, args []string) error {
			cmd.DisableAutoGenTag = true

```

Reply:
I set this via `cmd.Parent().Root().DisableAutoGenTag = true` so I didn't have to set it in individual commands.